src/model/attendance-board.test.ts

```diff
@@ -0,0 +1,38 @@
+import { ID } from "./id";
+
```

Comment:
```suggestion
import { Subject } from "./subject";

```
`Subject` の import が抜けています